### PR TITLE
fix(deps): use google-gax v3.5.2

### PR DESCRIPTION
**Pull request by alexander-fenster**
- fix(deps): use google-gax v3.5.2
- fix(deps): use google-gax v3.5.2

### CODE REVIEW

File: packages/gcf-utils/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "express": "^4.18.1",
     "gaxios": "^5.0.1",
     "get-stream": "^6.0.1",
-    "google-gax": "^3.3.1",
+    "google-gax": "^3.5.2"
```
Comment (chingor13) [inline packages/gcf-utils/package.json:44]: looks like we're missing a trailing comma
```suggestion
    "google-gax": "^3.5.2",
```

File: packages/secret-rotator/package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "@types/sinon": "^10.0.13",
     "c8": "^7.12.0",
     "gaxios": "^5.0.1",
-    "google-gax": "^3.4.0",
+    "google-gax": "^3.5.2"
```
Comment (chingor13) [inline packages/secret-rotator/package.json:38]:
```suggestion
    "google-gax": "^3.5.2",
```

**Review by chingor13 — CHANGES_REQUESTED** [review]

**Review by sofisl — APPROVED** [review]

**Review by sofisl — APPROVED** [review]

**Review by sofisl — APPROVED** [review]

**Review by sofisl — APPROVED** [review]

**Review by sofisl — APPROVED** [review]

**Comment by tmatsuo** [comment]
Our test and deployment scripts run `npm ci`.

Maybe you can run `npm update google-gax --save` to update the package-lock.json too.

**Comment by alexander-fenster** [comment]
Updated lockfiles. This was a mass change across all repositories that depend on `google-gax`, it may not be that important for `repo-automation-bots`, but let's still have it here as well if CI passes.

**Review by tmatsuo — APPROVED** [review]

**Review by chingor13 — APPROVED** [review]

**Pull request merged**